planner_configs:
  SBLkConfigDefault:
    type: geometric::SBL
  ESTkConfigDefault:
    type: geometric::EST
  LBKPIECEkConfigDefault:
    type: geometric::LBKPIECE
  BKPIECEkConfigDefault:
    type: geometric::BKPIECE
  KPIECEkConfigDefault:
    type: geometric::KPIECE
  RRTkConfigDefault:
    type: geometric::RRT
  RRTConnectkConfigDefault:
    type: geometric::RRTConnect
  RRTstarkConfigDefault:
    type: geometric::RRTstar
  TRRTkConfigDefault:
    type: geometric::TRRT
  PRMkConfigDefault:
    type: geometric::PRM
  PRMstarkConfigDefault:
    type: geometric::PRMstar
right_arm:
  planner_configs:
    - SBLkConfigDefault
    - ESTkConfigDefault
    - LBKPIECEkConfigDefault
    - BKPIECEkConfigDefault
    - KPIECEkConfigDefault
    - RRTkConfigDefault
    - RRTConnectkConfigDefault
    - RRTstarkConfigDefault
    - TRRTkConfigDefault
    - PRMkConfigDefault
    - PRMstarkConfigDefault
  projection_evaluator: joints(pan_joint,right_arm_wrist_yaw_joint)
  longest_valid_segment_fraction: 0.05
left_arm:
  planner_configs:
    - SBLkConfigDefault
    - ESTkConfigDefault
    - LBKPIECEkConfigDefault
    - BKPIECEkConfigDefault
    - KPIECEkConfigDefault
    - RRTkConfigDefault
    - RRTConnectkConfigDefault
    - RRTstarkConfigDefault
    - TRRTkConfigDefault
    - PRMkConfigDefault
    - PRMstarkConfigDefault
  projection_evaluator: joints(pan_joint,left_arm_wrist_tilt_joint)
  longest_valid_segment_fraction: 0.05
right_gripper:
  planner_configs:
    - SBLkConfigDefault
    - ESTkConfigDefault
    - LBKPIECEkConfigDefault
    - BKPIECEkConfigDefault
    - KPIECEkConfigDefault
    - RRTkConfigDefault
    - RRTConnectkConfigDefault
    - RRTstarkConfigDefault
    - TRRTkConfigDefault
    - PRMkConfigDefault
    - PRMstarkConfigDefault
left_gripper:
  planner_configs:
    - SBLkConfigDefault
    - ESTkConfigDefault
    - LBKPIECEkConfigDefault
    - BKPIECEkConfigDefault
    - KPIECEkConfigDefault
    - RRTkConfigDefault
    - RRTConnectkConfigDefault
    - RRTstarkConfigDefault
    - TRRTkConfigDefault
    - PRMkConfigDefault
    - PRMstarkConfigDefault
